package.json

```diff
@@ -95,38 +95,38 @@
     "babel-plugin-transform-es2015-modules-strip": "^0.1.1",
     "bundlesize": "^0.16.0",
     "clean-css-cli": "^4.1.11",
-    "cross-env": "^5.1.3",
+    "cross-env": "^5.1.4",
     "eslint": "^4.18.2",
     "eslint-plugin-compat": "^2.2.0",
     "glob": "^7.1.2",
     "htmllint-cli": "^0.0.6",
     "jsunitsaucelabs": "^1.3.0",
     "karma": "^2.0.0",
     "karma-chrome-launcher": "^2.2.0",
-    "karma-coverage-istanbul-reporter": "^1.4.1",
+    "karma-coverage-istanbul-reporter": "^1.4.2",
     "karma-detect-browsers": "^2.2.6",
     "karma-firefox-launcher": "^1.1.0",
     "karma-qunit": "^1.2.1",
     "karma-sinon": "^1.0.5",
     "node-sass": "^4.7.2",
-    "nodemon": "^1.17.1",
+    "nodemon": "^1.17.2",
     "npm-run-all": "^4.1.2",
-    "popper.js": "^1.12.9",
+    "popper.js": "^1.14.0",
     "postcss-cli": "^5.0.0",
     "qunitjs": "^2.4.1",
-    "rollup": "^0.56.4",
+    "rollup": "^0.56.5",
     "rollup-plugin-babel": "4.0.0-beta.0",
-    "rollup-plugin-node-resolve": "^3.0.3",
+    "rollup-plugin-node-resolve": "^3.2.0",
     "shelljs": "^0.8.1",
     "shx": "^0.2.2",
-    "sinon": "^4.4.3",
+    "sinon": "^4.4.5",
```

Comment:
should we upgrade to their new major release Sinon 5 ? Because currently we haven't unit tests which use Sinon (but it'll happen soon)